src/helpers/configuration-validators.ts

```diff
@@ -1,18 +1,20 @@
 import { ErrorCode, PurchasesError } from "../entities/errors";
 import { SDK_HEADERS } from "../networking/http-client";
+import {
+  isPaddleApiKey,
+  isRCTestStoreApiKey,
+  isWebBillingApiKey,
+} from "./api-key-helper";
 
 export function validateApiKey(apiKey: string) {
-  const rc_api_key_regex = /^rcb_[a-zA-Z0-9_.-]+$/;
-  const paddle_api_key_regex = /^pdl_[a-zA-Z0-9_.-]+$/;
-  const rc_test_store_api_key_regex = /^test_[a-zA-Z0-9_.-]+$/;
   if (
-    !rc_api_key_regex.test(apiKey) &&
-    !paddle_api_key_regex.test(apiKey) &&
-    !rc_test_store_api_key_regex.test(apiKey)
+    !isWebBillingApiKey(apiKey) &&
+    !isPaddleApiKey(apiKey) &&
+    !isRCTestStoreApiKey(apiKey)
   ) {
     throw new PurchasesError(
       ErrorCode.InvalidCredentialsError,
-      "Invalid API key. Use your Web Billing or Paddle API key.",
+      "Invalid API key. Use your Web Billing API key.",
```

Comment:
@tonidero should we indicate that **SimulatedStore** API keys are supported, or is it still WIP?

Reply:
Still WIP, so better not to add it here for now. Thanks for checking!